kompose.spec

```diff
@@ -52,8 +52,8 @@
 %global testflags -race -cover -v
 
 Name:           kompose
-Version:        0.4.0
-Release:        0.1.git%{shortcommit}%{?dist}
```

Comment:
can you remove the shortcommit in a 2nd commit? I want this to be a clear change we decided to do and not something that could be confused as a mistake.